moped-editor/src/views/projects/projectView/ProjectStatusBadge.js

```diff
@@ -6,19 +6,22 @@ import PauseCircleOutlineOutlinedIcon from "@material-ui/icons/PauseCircleOutlin
 import RemoveCircleOutlineOutlinedIcon from "@material-ui/icons/RemoveCircleOutlineOutlined";
 import PlayCircleOutlineOutlinedIcon from "@material-ui/icons/PlayCircleOutlineOutlined";
 import CheckCircleOutlineOutlinedIcon from "@material-ui/icons/CheckCircleOutlineOutlined";
+import HelpOutlineIcon from "@material-ui/icons/HelpOutline";
 
 import { makeStyles } from "@material-ui/styles";
 import clsx from "clsx";
 
-const defaultIcon = PlayCircleOutlineOutlinedIcon;
+const defaultIcon = HelpOutlineIcon;
+const defaultLabel = "Unknown";
```

Comment:
I'm curious what others think about the question mark icon and "Unknown" label for default badge, which appears if a project does not have a current phase.

Also considered an exclamation point icon. For the label, maybe "Unknown status" or "No status" would be more clear.

<img width="614" alt="Screen Shot 2022-11-12 at 11 16 28 AM" src="https://user-images.githubusercontent.com/14793120/201483644-69a8d11e-79a5-40a8-84f8-6187d49ac5bc.png">

Reply:
Agreed. It's also not super obvious that the "status" magically comes from the Phase table on the Timeline tab....

In addition to reworking that status badge language, maybe we should rename the "Status" column on the project list to "Phase"?

I also thought it would be cool to add a tooltip to the badge itself that shows the definition of the phase, or if it's "unknown" it would say "You need to add a current phase to this project"...

Reply:
cc @patrickm02L

Reply:
Unknown makes sense to me, and the badge with question mark works.

Reply:
I think John's thoughts on the tooltip are insightful. Since we're having this conversation at all, I think it's safe to say there is going to be confusion about what Unknown means, what is "?" about it, and how did it happen. In the space of a badge, I don't expect there to be a single word or symbol that can convey all those things well.

From this, I think John's thoughts on a tooltip that have more room to guide the user and educate them will lead to more statuses getting created than any word-choice or color-treatment might. 

All that to say, I'm fully on board with a tool tip, and John's ideas on what to populate it with sound like an awesome place to start.